lib/identity_doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -116,7 +116,7 @@ def create_response_info
             transaction_reason_code: transaction_reason_code,
             doc_auth_result: doc_auth_result,
             processed_alerts: alerts,
-            alert_failure_count: alerts[:failed].length,
+            alert_failure_count: alerts[:failed]&.count.to_i,
```

Comment:
What was the reason for this change, and the similar one below? I don't see any path where `alerts[:failed]` could not exist or be anything other than an array, and the tests appear to pass with the previous implementation?

Reply:
I think this was a consistency thing I wanted to clean up. It's just null safety thing, but I agree with the way things are set up now alerts should at worst be an empty array.